docs/editors/vim.md

```diff
@@ -58,7 +58,7 @@ following settings.
 Plug 'neoclide/coc.nvim', {'branch': 'release'}
 
 " Configuration for vim-scala
-au BufRead,BufNewFile *.sbt set filetype=scala
+au BufRead,BufNewFile *.sbt,*.sc set filetype=scala
```

Comment:
Good catch. Can we actually change the note here to explain a bit as to why it is being done. Something like _Necessary for Vim to recognize sbt, script, and worksheet files as Scala._

Also, are then able to move this entire chunk down to the recommended mappings file? It's a bit better fitting there since this isn't related to a specific plugin, but rather a general setting.